avro-format.md

```diff
@@ -0,0 +1,176 @@
+# Avro Event Format for CloudEvents
+
+## Abstract
+
+The Avro Format for CloudEvents defines how events attributes are expressed in the [Avro 1.9.0 Specification][avro-spec].
+
+## Status of this document
+
+This document is a working draft.
+
+## Table of Contents
+
+1. [Introduction](#1-introduction)
+2. [Attributes](#2-attributes)
+3. [Examples](#3-examples)
+
+## 1. Introduction
+
+[CloudEvents][ce] is a standardized and transport-neutral definition of the
+structure and metadata description of events. This specification defines how the
+elements defined in the CloudEvents specification are to be represented in the
+[Avro 1.9.0][avro-primitives].
+
+The [Attributes](#2-attributes) section describes the naming conventions and
+data type mappings for CloudEvents attributes for use as Avro message
+properties.
+
+This specification does not define an envelope format. The Avro type system's
+intent is primarily to provide a consistent type system for Avro itself and not
+for message payloads.
+
+### 1.1. Conformance
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119][rfc2119].
+
+## 2. Attributes
+
+This section defines how CloudEvents attributes are mapped to the Avro
+type-system. This specification explicitly maps each attribute.
+
+### 2.1 Type System Mapping
+
+The CloudEvents type system MUST be mapped to Avro types as follows.
+
+| CloudEvents   | Avro                                                                   |
+| ------------- | ---------------------------------------------------------------------- |
+| Integer       | [int][avro-primitives]                                                 |
+| String        | [string][avro-primitives]                                              |
+| Binary        | [bytes][avro-primitives]                                               |
+| Map           | [map][avro-primitives]                                                 |
+| Any           | See [2.2](#22-mapping-any-typed-attributes)                            |
```

Comment:
I like that the order is the same as what's in the spec... but ANY has been moved to the end now  :-)